readme.md

```diff
@@ -3,30 +3,47 @@
 # AMP Service worker
 AMP service worker is a service worker library for your AMP pages.
 
-The project aims to bring network resiliency and other network related optimizations to any AMP page which is being controlled by this service worker.
-Core use cases:
+**Purpose:** The project aims to bring network resiliency and other network related optimizations to any AMP page which is being controlled by this service worker.
+
+**Core use cases:**  
 - Cache AMP scripts with a `stale-while-revalidate` strategy for a longer duration that http-headers.
 - Cache valid visited AMP documents, and serve only in case of flaky network conditions.
 - Cache assets which are critical to a page with a given strategy.
-- Prefetch outgoing links from an AMP page.
 - Cache an offline page in order to show when a user navigates to a page which was previously not visited.
+- Prefetch outgoing links from an AMP page.
```

Comment:
The mixed usage of page and document is slightly confusing. I'd choose one a day stick with it.